src/main/resources/assets/mmorpg/lang/zh_cn.json

```diff
@@ -1,113 +1,98 @@
+	"item.mmorpg.player_levelup": "玩家升级令牌",
+	"item.mmorpg.loot_bag": "装备战利品袋",
+	"item.mmorpg.map_bag": "地图袋",
+	"item.mmorpg.currency_bag": "通货袋",
+	"item.mmorpg.newbie_gear_bag": "新手装备袋",
+	"item.mmorpg.tutorial_book": "挖矿与砍杀教程",
+	"item.mmorpg.map_back_portal": "地图回返珍珠",
+	"item.mmorpg.level_nearest_entity": "提升邻近实体的等级",
+	"item.mmorpg.increase_rarity_nearest_entity": "提升邻近实体的品级",
+	"item.mmorpg.mat.infused_iron": "注源铁锭",
+	"item.mmorpg.mat.crystallized_essence": "晶化源质",
+	"item.mmorpg.mat.golden_orb": "灿金宝珠",
+	"item.mmorpg.mat.mythic_essence": "神话源质",
+	"item.mmorpg.awaken_runeword": "觉醒符文之语",
+	"item.mmorpg.identify_tome": "鉴定卷轴",
+	"item.mmorpg.identify_tome_plus": "鉴定卷册",
+	"item.mmorpg.teleport_scroll": "返回卷轴",
+	"item.mmorpg.prefix": "前缀",
+	"item.mmorpg.suffix": "后缀",
+	"block.mmorpg.scrabble": "谜语箱",
+	"block.mmorpg.map_chest": "地图箱子",
+	"mmorpg.tooltip.aoe_spell_explosion": "释放一枚元素弹射物，命中后造成范围伤害。",
+	"mmorpg.tooltip.single_target_spell_bolt": "释放一枚元素弹射物，对单个目标造成伤害。",
+	"mmorpg.tooltip.spell_instant_heal": "立即治疗施法者。",
+	"mmorpg.tooltip.aoe_bomb_spell": "留下一枚炸弹，之后造成范围爆炸伤害。",
+	"mmorpg.tooltip.spell_self_regen": "持续治疗施法者的生命值。",
+	"mmorpg.tooltip.aoe_spell_nova": "伤害你周围的所有目标。",
+	"mmorpg.tooltip.ele_bonus_atk_dmg_spell": "给予增益效果，消耗法力造成额外伤害。",
+	"item.mmorpg.capacitor0": "凡品能效珠",
+	"item.mmorpg.capacitor1": "下品能效珠",
+	"item.mmorpg.capacitor2": "中品能效珠",
+	"item.mmorpg.capacitor3": "上品能效珠",
+	"item.mmorpg.capacitor4": "精品能效珠",
+	"item.mmorpg.capacitor5": "玄品能效珠",
+	"death.attack.mmorpg.custom_damage": "%1$s被残暴地杀害了",
+	"mmorpg.name.map": "冒险地图",
+	"mmorpg.tooltip.put_in_mapdevice": "右击地图装置使用",
+	"mmorpg.tooltip.press_shift_more_info": "[Shift]：信息，[Alt]：描述，[Ctrl]：附魔",
+	"mmorpg.tooltip.affix_rarity_lootbonus": "战利品词缀品级增益",
+	"curios.identifier.salvage_bag": "回收袋",
+	"curios.identifier.bracelet": "手镯",
+	"entity.mmorpg.trader.none": "地牢商人",
+	"block.mmorpg.dungeon_portal": "地牢传送门",
+	"entity.mmorpg.spirit_wolf_pet": "召唤猎狼",
+	"entity.mmorpg.zombie_pet": "召唤僵尸",
+	"entity.mmorpg.skeleton_pet": "召唤骷髅",
+	"entity.mmorpg.archon_pet": "召唤执政官",
+	"mmorpg.mob_affix.accurate": "精准的",
+	"mmorpg.mob_affix.cold": "寒冷的",
+	"mmorpg.mob_affix.evasive": "闪避的",
+	"mmorpg.mob_affix.flaming": "火烈的",
+	"mmorpg.mob_affix.lightning": "雷电的",
+	"mmorpg.mob_affix.of_conductivity": "之导电",
+	"mmorpg.mob_affix.of_harsh_winter": "之严冬",
+	"mmorpg.mob_affix.of_leeching": "之吸血",
+	"mmorpg.mob_affix.of_liquid_death": "之死亡之水",
+	"mmorpg.mob_affix.of_mountains": "之山脉",
+	"mmorpg.mob_affix.of_raging_volcano": "之火山喷涌",
+	"mmorpg.mob_affix.of_retribution": "之报复",
+	"mmorpg.mob_affix.of_rock_skin": "之石肤",
+	"mmorpg.mob_affix.of_scalding": "之滚烫",
+	"mmorpg.mob_affix.of_the_cobra": "之眼镜蛇",
+	"mmorpg.mob_affix.of_the_sniper": "之狙击",
+	"mmorpg.mob_affix.of_thunder_strikes": "之闪电打击",
+	"mmorpg.mob_affix.of_toxic_fumes": "之毒气",
+	"mmorpg.mob_affix.of_venom_shock": "之毒痹",
+	"mmorpg.mob_affix.resistant": "抗性的",
+	"mmorpg.mob_affix.venom": "毒牙的",
```

Comment:
The old translation doesn't use the same way to translate the affix of mob and item, that is weird. So I change these for the consistency.